src/components/Button.js

```diff
@@ -13,7 +13,12 @@ const Button = (props) => {
     classes += 'basis-50';
   }
   return (
-    <button className={classes} type="button">
+    <button
+      className={classes}
+      // eslint-disable-next-line react/destructuring-assignment
+      onClick={props.onClick}
```

Comment:
Instead of disabling this rule, we can add `onClick` to the props that is destructured above.

```js
  const {
    btnLabel, color, wide, onClick,
  } = props;
```
Then use directly here as 
```js
onClick={onClick}
```